docs/high-avaliability/prometheus-HA.md

```diff
@@ -0,0 +1,19 @@
+# Using Promscale with Prometheus in High Availability
+
+Promscale supports running Prometheus in High Availability (HA). This means that more than one Prometheus instances can run in parallel. Promscale implements leader election to support Prometheus HA mode through pg_advisory_lock.
+
+## Leader-election method via pg_advisory_lock
+
+PostgreSQL provides a [pg_advisory_lock](https://www.postgresql.org/docs/current/explicit-locking.html#ADVISORY-LOCKS) that locks a application-defined resource based on an ID. Promscale makes use of this lock when running multiple promescale instances in parallel in order to choose one leader.
+
+![Promscale architecture for Prometheus in HA](https://github.com/Harkishen-Singh/promscale/blob/docs_HA_promscale/docs/assets/promscale-HA-arch.png?raw=true)
```

Comment:
The image neeeds to be improved. The export seems to be of low quality, there are missing arrows from prometheus to promscale in cluster 2 and there is a hand symbol in the middle of the image for some reason